pkg/tfbridge/schema.go

```diff
@@ -1006,9 +1006,9 @@ func MakeTerraformOutput(p shim.Provider, v interface{},
 		switch val.Kind() {
 		case reflect.Bool:
 			return resource.NewBoolProperty(val.Bool())
-		case reflect.Int:
+		case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
 			return resource.NewNumberProperty(float64(val.Int()))
```

Comment:
// Int returns v's underlying value, as an int64.
// It panics if v's Kind is not Int, Int8, Int16, Int32, or Int64.
func (v Value) Int() int64 {

This sounds good, it's th largest of them.